projects/packages/forms/composer.json

```
@@ -3,11 +3,10 @@
 	"description": "Jetpack Forms",
 	"type": "jetpack-library",
 	"license": "GPL-2.0-or-later",
-	"require": {},
 	"require-dev": {
 		"yoast/phpunit-polyfills": "1.0.4",
 		"automattic/jetpack-changelogger": "@dev",
-		"automattic/wordbless": "dev-master"
+		"automattic/wordbless": "^0.4.1"
```

Should we use `@dev` here as well?

```suggestion
		"automattic/wordbless": "@dev"
```